sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -111,6 +111,7 @@ def language_result(language):
     return DetectLanguageResult(
         id=language.id,
         detected_languages=[DetectedLanguage._from_generated(l) for l in language.detected_languages],  # pylint: disable=protected-access
+        primary_language=DetectedLanguage._from_generated(language.detected_languages[0]),  # pylint: disable=protected-access
```

Comment:
can `language.detected_languages` be null?

Reply:
It's listed as required in the swagger, so no. If it can't detect the language it will still return an object like {'name': '(Unknown)', 'iso6391_name': '(Unknown)', 'score': 0.0}